dev/drivers/scripts/plots/wafs/jevs_wafs_atmos_plots.sh

```diff
@@ -36,28 +35,27 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 ############################################################
 # environment variables set
 ############################################################
-export envir=dev
+export envir=prod
 
 export NET=evs
 export STEP=plots
 export COMPONENT=wafs
 export RUN=atmos
 export VERIF_CASE=grid2grid
 
-export COMIN=${COMIN:-/lfs/h2/emc/vpppg/noscrub/${USER}/${NET}/$evs_ver_2d}
+#export COMIN=${COMIN:-/lfs/h2/emc/vpppg/noscrub/${USER}/${NET}/$evs_ver_2d}
 export COMROOT=/lfs/h2/emc/ptmp/$USER
 
-export USH_DIR=$HOMEevs/ush/$COMPONENT
```

Comment:
@YaliMao-NOAA I think we need to remove the comment here, as we need to have COMIN set in the driver.  Otherwise it would look in the production directory.

Also, do we need COMROOT set here?

Reply:
This driver is my test version, either COMIN or COMROOT can be set for COMIN.
But I will change as you suggested.